resources/themes/cura-light/icons/group_48px.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Please don't name vector images as to how many pixels they are meant to cover. They will not cover that many pixels on many screens. There are no pixels involved in the file, and different SVG renderers will have different pixels since they all do anti-aliasing differently.